### PR TITLE
feat: documentation updates

### PR DESCRIPTION
**Pull request by tanvibhakta**
## Description

Fixes https://github.com/appsmithorg/appsmith/issues/26246, https://github.com/appsmithorg/appsmith/issues/26493
Updates spinner documentation 
Updates types for callout's children.
Depends on https://github.com/appsmithorg/appsmith/pull/26498

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Chore (housekeeping or task changes that don't impact user perception)
- This change requires a documentation update

- Manual on storybook 
- Manual on main repo
- Jest
- Cypress

### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: packages/design-system/plop-templates/component.mdx.hbs [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import { Canvas, Meta } from '@storybook/blocks';
```
Comment (albinAppsmith) [inline packages/design-system/plop-templates/component.mdx.hbs:1]: Along with this, you need to update the generate script also

**Review by albinAppsmith — APPROVED** [review]

**Comment by tanvibhakta** [comment]
tests have passed: https://github.com/appsmithorg/appsmith/pull/26498#issuecomment-1685937506

**Comment by albinAppsmith** [comment]
Is the highlighted part rendering correctly?

URL: https://design-system-git-chore-documentation-updates-get-appsmith.vercel.app/?path=/docs/design-system-collapsible--documentation
Section: Examples

Screenshot 2023-08-22 at 10 07 37 AM

**Comment by albinAppsmith** [comment]
Screenshot 2023-08-22 at 10 08 39 AM

Spinner anatomy image not loading

**Comment by tanvibhakta** [comment]
Spinner images don't exist yet, I've commented them out, and I've change the comment on Collapsible as well.

**Comment by albinAppsmith** [comment]
Please create a task for the spinner anatomy image for @vasanthappsmith

**Review by albinAppsmith — APPROVED** [review]

**Comment by tanvibhakta** [comment]
Done: https://github.com/appsmithorg/appsmith/issues/26553

**Pull request merged**